July-Morning/assignment4/disjoint_set.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef DISJOINT_SET_HPP_
+#define DISJOINT_SET__HPP_
```

Comment:
Not that important, but the `#ifndef` and `#define` have different strings so it is probably not going to do what you expect :)